collectors/proc.plugin/README.md

```diff
@@ -57,12 +60,19 @@ Hopefully, the Linux kernel provides many metrics that can provide deep insights
     Of course, for newer disk technologies (like fusion cards) that are capable to execute multiple commands in parallel, this metric is just meaningless.
 -   **Average I/O operation time (ms)**
     The average time for I/O requests issued to the device to be served. This includes the time spent by the requests in queue and the time spent servicing them.
+-   **Average I/O operation time for extended operations (ms)**
+    The average time for extended I/O requests issued to the device to be served. This includes the time spent by the requests in queue and the time spent servicing them.
 -   **Average I/O operation size (kb)**
     The average amount of data of the completed I/O operations.
+-   **Average amount of discarded data (kb)**
+    The average amount of data of the completed discard operations.
 -   **Average Service Time (ms)**
     The average service time for completed I/O operations. This metric is calculated using the total busy time of the disk and the number of completed operations. If the disk is able to execute multiple parallel operations the reporting average service time will be misleading.
+-   **Average Service Time for extended I/O operations (ms)**
+    The average service time for completed extended I/O operations.
 -   **Merged I/O operations/s**
     The Linux kernel is capable of merging I/O operations. So, if two requests to read data from the disk are adjacent, the Linux kernel may merge them to one before giving them to disk. This metric measures the number of operations that have been merged by the Linux kernel.
+-   **Merged discard operations/s**
 -   **Total I/O time**
```

Comment:
I did not understand these two formats together, I think you wanted either to remove `Total I/O time` or you wanted to convert it for a simple text, please, can you explain me this?

Reply:
Metric names are in bold text, regular text is for explanations. "Merged discard operations/s" doesn't have an explanation, the name says it all.

Reply:
Yeah, indeed, imaging do not know what _Merged discard operations_ is in 2021 😄

Reply:
Well, the explanation for merged operations, in general, is under "Merged I/O operations/s". I didn't want to repeat it again word for word.